quirks/websites-with-shared-credential-backends.json

```diff
@@ -288,6 +288,30 @@
         "optumrx.com",
         "optum.com"
     ],
+    [
+        "pinterest.com",
+        "pinterest.ca",
+        "pinterest.co.uk",
+        "pinterest.fr",
+        "pinterest.de",
+        "pinterest.es",
+        "pin.it",
```

Comment:
I'd say similar logic applies here; `pin.it` is a redirect and should be removed from this PR.